### DISCUSSION TITLE
Candlestick bar

### DISCUSSION THREAD

**Discussion by Sharvinqw** [apexcharts/apexcharts.js#3752] [Q&A]
I am using the candlestick vue3 apexcharts,
I would like to change the border radius of the bar in the chart.
![Screenshot 2023-04-03 at 12 55 30 PM](https://user-images.githubusercontent.com/128598605/229461260-65db7f3d-2c3e-48a9-9699-bb6cc23e2ba6.png)

**Comment by janstieler** [comment]
I don't think this is possible because there is no svg equivalent to css border-radius and in the documentation is no option for this declared.
You could wrote your own function but I think this is really scary.
